views/beers.hbs

```diff
@@ -0,0 +1,7 @@
+<h1>Beers</h1>
+{{#each beersFromApi}}
+<h2>{{this.name}}</h2>
+<img src="{{this.image_url}}" alt="beer">
+<h3>{{this.tagline}}</h3>
+<p>{{this.description}}</p>
```

Comment:
If you have some time, try experimenting with Partials. It is a great way to reuse code in different hbs files. See here we have `image`, `name`, `tagline` and `description` and they are used again in case of a random beer. These 4 could go into a Partial.